product_docs/docs/pgd/4/rel_notes/pgd_4.3.5_rel_notes.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.5"
+navTitle: "Version 4.3.5"
+redirects:
+  - /pgd/latest/bdr/release_notes/bdr4.3.5_rel_notes/
+---
+
+Released: 14 May 2024
+
+EDB Postgres Distributed version 4.3.5 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.10, 15.5 and later.
+!!!
+
+| Component | Version | Type         | Description                                                                                        |
+|-----------|---------|--------------|----------------------------------------------------------------------------------------------------|
+| HARP      | 2.4.1   | Change       | Routine security library upgrades.                                                                 |
+| BDR       | 5.5.0   | Enhancement  | Log completion of an extension upgrade.                                                            |
```

Comment:
@djw-m did we get rid of the support tickets column? thought that was helpful for customers to reference - Mastercard will be looking for this attach/detach fix